astropy/io/fits/fitsrec.py

```diff
@@ -848,6 +848,17 @@ def _convert_ascii(self, column, field):
         null_fill = encode_ascii(str(ASCIITNULL).rjust(format.width))
         dummy = np.where(np.char.strip(dummy) == nullval, null_fill, dummy)
 
+        # But if the nullval itself is a blank string and it's an int column,
+        # it must be converted to a 0 before being loaded into the numpy array.
+        # The nullval for an empty string is the string 'None'.
```

Comment:
Is this generally true or just in this case? I can't find any mention of it in the FITS standard and I haven't found any such specification in astropy.io.fits, but I just did a quick search.